labgrid/remote/exporter.py

```diff
@@ -35,6 +35,17 @@ class BrokenResourceError(ExporterError):
     pass
 
 
+def log_subprocess_kernel_stack(logger, child):
+    if child.poll() is not None:  # nothing to check if no longer running
+        return
+    try:
+        with open('/proc/{}/stack'.format(child.pid), 'r') as f:
+            stack = f.read()
+            stack = stack.strip()
+    except PermissionError:
+        return
+    logger.info("current kernel stack of {} is:\n".format(child.args)+stack)
```

Comment:
Nit: % style formatting should be used in logging functions.

Reply:
Yes, but this is very far from performance critical code. ;)

Reply:
Agreed, but this will be another pylint warning.